Packs/CommonScripts/Scripts/ExtractIndicatorsFromTextFile/ExtractIndicatorsFromTextFile.py

```diff
@@ -19,10 +19,10 @@
 with open(filePath, mode='r') as f:
     data = f.read(maxFileSize)
     try:
-        data = data.decode('unicode_escape').encode('utf-8')
+        data = data.decode('iso-8859-1', 'unicode_escape').encode('utf-8')
```

Comment:
Seems that UTF-8 is a wider option, we can only add more options, not reduce them